lib/miq_environment.rb

```diff
@@ -14,6 +14,12 @@ def self.supports_apache?
       @supports_apache = self.is_appliance? && self.supports_command?('apachectl')
     end
 
+    def self.supports_systemd?
+      return true
```

Comment:
temporary?

Reply:
Ah yeah, I wasn't running on an appliance so I wanted to shortcut the `is_appliance` check...removed